images/10-qemuguestagent/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM alpine:3.7
+# FROM amd64=skip arm64=skip
+
+RUN apk add --no-cache qemu-guest-agent
+
+ENTRYPOINT ["/usr/bin/ros", "entrypoint"]
+
+RUN addgroup -g 1100 rancher && \
```

Comment:
Not sure what use these scripts have for qemu-guest-agent.
If you need these, please add some comments, so that other people can also maintain it.